### PR TITLE
Travis CI support 

### LINKED ISSUE #4: Test on Travis

### PR DESCRIPTION
**Pull request by frewsxcv**
Fixes #4

### CODE REVIEW

**Comment by frewsxcv** [comment]
Reviewable

**Comment by streeter** [comment]
LGTM

File: runtests.py [hunk 1]
```diff
@@ -2,12 +2,28 @@
...
 # Django must be set up before we import our libraries and run our tests
 
+import os
 import sys
 
 import django
 from django.conf import settings
 
 
+if os.environ.get('TEST_WITH_REDIS'):
```
Comment (meastman) [inline runtests.py]: This will always be true-ish for travis since it's a non-empty string.

Comment (streeter) [inline runtests.py]: Oh, we'd need to convert it to an int then?

Comment (meastman) [inline runtests.py]: Yeah; I'd suggest `if int(os.environ.get('TEST_WITH_REDIS', 0))`

Comment (meastman) [inline runtests.py]: Another alternative would be to have one of the `env` values above be `TEST_WITH_REDIS=` (empty string)

**Pull request merged**